app/controllers/pages_controller.rb

```diff
@@ -3,4 +3,10 @@ class PagesController < ApplicationController
 
   def home
   end
+
+  def about
+  end
+
+  def my_profile
```

Comment:
Wouldn't this be part of the user?

Reply:
Yep I think you're right, I completely forgot about devise.